src/model/trust.page.model.ts

```diff
@@ -35,13 +36,42 @@ interface TrustHistoricalBeneficialOwnerFormCommon {
   endDateYear: string;
 }
 
+type IndividualTrusteesFormCommon = {
+  trusteeId?: string,
+  type: RoleWithinTrustType,
+  forename: string,
+  surname: string,
+  date_of_birth_day: string,
+  date_of_birth_month: string,
+  date_of_birth_year: string,
+  nationality: string,
+  second_nationality?: string,
+  property_name?: string,
+  address_line1: string,
+  address_line2: string,
+  city: string,
+  county: string,
+  country: string,
+  postal_code: string,
```

Comment:
After looking back at mine I think:
-`property_name?: string,` should not be optional. As it links to premise which is essential the house name or number(wierd naming)
- `county: string,` I think should be optional as marked so on the form. (I updated this myself)
- `postal_code: string,`  I think should be optional as it's marked as "if applicable" on the form. (I updated this myself)
Here is the link to the prototype for you too see - https://roe-v13.herokuapp.com/register/trusts/trust-individual

These changes should be reflected in the `trust.model.ts` too

Happy to approve after this

Reply:
thanks missed it